kedro-org/kedro-viz
# Display hosted URL in CLI while launching kedro viz

## Description

Resolves #1638 

## Development notes

* Added print statement to display the url on which kedro viz has started running
* Fixed dependency conflict of s3fs with kedro-datasets

## QA notes
* Install package using `pip install -e package`
* `cd demo-project`
* Run `kedro viz`
* Url information as below should be displayed

![image](https://github.com/kedro-org/kedro-viz/assets/87735534/e211fd31-b55e-4fb9-b15e-db404fa33bf5)


## Checklist
